pyqtgraph/graphicsItems/tests/test_ErrorBarItem.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from pyqtgraph.Qt import QtGui, QtCore
+import pyqtgraph as pg
+import numpy as np
+
+app = pg.mkQApp()
+
+
+def test_errorbaritem_defer_data():
```

Comment:
Make camelCase ?  `test_errorBarItemDeferData` perhaps?

Reply:
Seems to be mixed convention in the code base. One thing that seems pretty consistent is `test_ClassName_what_this_test_does`. So `test_ErrorBarItem_defer_data`?

Reply:
Given we don't have any consistent naming scheme, I would write whatever you want in haha; renaming to a consistent convention should happen at some-point, but not with this PR :)